code/modules/response_team/ert.dm

```diff
@@ -0,0 +1,369 @@
+// ERTs
+
+#define ERT_TYPE_AMBER		1
+#define ERT_TYPE_RED		2
+#define ERT_TYPE_GAMMA		3
+
+/datum/game_mode
+	var/list/datum/mind/ert = list()
+
+var/list/response_team_members = list()
+var/responseteam_age = 21 // Minimum account age to play as an ERT member
+var/datum/response_team/active_team = null
+var/send_emergency_team = 0
+var/ert_request_answered = 0
+
+/client/proc/response_team()
+	set name = "Dispatch CentComm Response Team"
+	set category = "Event"
+	set desc = "Send an CentComm response team to the station."
+
+	if(!check_rights(R_EVENT))
+		return
+
+	if(!ticker)
+		to_chat(usr, "<span class='warning'>The game hasn't started yet!</span>")
+		return
+
+	if(ticker.current_state == 1)
+		to_chat(usr, "<span class='warning'>The round hasn't started yet!</span>")
+		return
+
+	if(send_emergency_team)
+		to_chat(usr, "<span class='warning'>Central Command has already dispatched an emergency response team!</span>")
+		return
+
+	var/datum/nano_module/ert_manager/E = new()
+	E.ui_interact(usr)
+
+
+/mob/dead/observer/proc/JoinResponseTeam()
+	if(!send_emergency_team)
+		to_chat(src, "No emergency response team is currently being sent.")
+		return 0
+
+	if(jobban_isbanned(src, ROLE_ERT))
+		to_chat(src, "<span class='warning'>You are jobbanned from the emergency reponse team!</span>")
+		return 0
+
+	var/player_age_check = check_client_age(client, responseteam_age)
+	if(player_age_check && config.use_age_restriction_for_antags)
+		to_chat(src, "<span class='warning'>This role is not yet available to you. You need to wait another [player_age_check] days.</span>")
+		return 0
+
+	if(cannotPossess(src))
+		to_chat(src, "<span class='boldnotice'>Upon using the antagHUD you forfeited the ability to join the round.</span>")
+		return 0
+
+	if(response_team_members.len > 6)
+		to_chat(src, "The emergency response team is already full!")
+		return 0
+
+	return 1
+
+/proc/trigger_armed_response_team(var/datum/response_team/response_team_type, commander_slots, security_slots, medical_slots, engineering_slots, janitor_slots, paranormal_slots, cyborg_slots)
+	response_team_members = list()
+	active_team = response_team_type
+	active_team.setSlots(commander_slots, security_slots, medical_slots, engineering_slots, janitor_slots, paranormal_slots, cyborg_slots)
+
+	send_emergency_team = 1
+	var/list/ert_candidates = pollCandidates("Join the Emergency Response Team?",, responseteam_age, 600, 1, role_playtime_requirements[ROLE_ERT])
+	if(!ert_candidates.len)
+		active_team.cannot_send_team()
+		send_emergency_team = 0
+		return 0
+
+	// Respawnable players get first dibs
+	for(var/mob/dead/observer/M in ert_candidates)
+		if(jobban_isbanned(M, ROLE_TRAITOR) || jobban_isbanned(M, "Security Officer") || jobban_isbanned(M, "Captain") || jobban_isbanned(M, "Cyborg"))
```

Comment:
As we have job bans for ERT - it seems odd to have you also ERT banned if you're banned from Tot/Sec/Captain/Cyborg.....

Reply:
Something that Spartan also brought up. This could be addressed, however I think it's more for an admin policy discussion and a separate PR.

Reply:
Can a Maintainer please weight in?

As we have a separate jobban for ERT, it seems funky it also extends the job bans for being job banned from other roles as well.